SPECS/metal3-baremetal-operator/metal3-baremetal-operator.spec

```diff
@@ -0,0 +1,41 @@
+Summary:        Bare Metal Operator implements a Kubernetes API for managing bare metal hosts
+Name:           metal3-baremetal-operator
+Version:        0.4.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            https://github.com/metal3-io/baremetal-operator
+# Created using ./generate-sources.sh
```

Comment:
Can this comment be expanded to say why generate-sources.sh is needed? It might be good for the spec file to survive as an independent entity (incase it is moved away from the script somehow. The source of the source code could be added (`# Source0:        https://github.com/metal3-io/baremetal-operator/archive/refs/tags/%%{version}.tar.gz`) and a comment (and maybe even a subset of the script) that makes clear that go modules are being packaged into it.

Reply:
Make sense. I will summarize the script and include the long tar command here.